build.sbt

```diff
@@ -207,14 +206,19 @@ lazy val docSettings = Seq(
   scalacOptions in (ScalaUnidoc, unidoc) ++= Seq(
     "-Xfatal-warnings",
     "-groups",
-    "-doc-source-url", scmInfo.value.get.browseUrl + "/tree/master€{FILE_PATH}.scala",
-    "-sourcepath", baseDirectory.in(LocalRootProject).value.getAbsolutePath,
+    "-doc-source-url",
+    scmInfo.value.get.browseUrl + "/tree/master€{FILE_PATH}.scala",
+    "-sourcepath",
+    baseDirectory.in(LocalRootProject).value.getAbsolutePath,
     "-diagrams"
-  ) ++ (if(priorTo2_13(scalaVersion.value)) Seq(
-    "-Yno-adapted-args",
-  ) else Seq(
-    "-Ymacro-annotations"
-  )),
+  ) ++ (if (priorTo2_13(scalaVersion.value))
+          Seq(
+            "-Yno-adapted-args",
+          )
+        else
+          Seq(
+            "-Ymacro-annotations"
+          )),
```

Comment:
This seems a bit excessive with respect to whitespace to me, but I don't know if this is something that's easily configurable?

Reply:
It just wants the closing bracket to be on the same indent as the opening one or on a single line.
We just need to put it on a single line the thing. Tho I'm not going to do it for all of them.